templates/vhost/_security.erb

```diff
@@ -5,6 +5,18 @@
 <% if @modsec_audit_log_destination -%>
   SecAuditLog "<%= @modsec_audit_log_destination %>"
 <% end -%>
+
+<% @modsec_before_crs.flatten.each do |rule| -%>
```

Comment:
Instead of emitting the rules in the vhost configuration file, can we instead put them in `${modsec_dir}/modsecurity_crs_15_customrules.conf` somehow, as the document you linked to suggests?